SConstruct

```diff
@@ -691,7 +691,7 @@ def runCommand( command ) :
 
 	command = commandEnv.subst( command )
 	sys.stderr.write( command + "\n" )
-	subprocess.check_call( command, shell=True, env=commandEnv["ENV"] )
+	return subprocess.check_output( command, shell=True, env=commandEnv["ENV"] ).decode()
```

Comment:
The decode is a bytes-to-string thing, right? Elsewhere, we've been doing that via the weirdly named `universal_newlines` argument to `check_output()`, which I think has a few benefits :

- It has some smarts about what encoding is in use
- When we finally ditch Python 3 we can replace it with the more explanatory `text` argument

Reply:
Fixed along with other uses of `decode` in https://github.com/GafferHQ/gaffer/pull/4524/commits/3084790bcb6a7f0df914d6a46269e98a102937e4